starknet_py/tests/e2e/docs/guide/test_resolving_proxies.py

```diff
@@ -32,34 +28,34 @@ async def test_resolving_proxies(
     contract = await Contract.from_address(address=address, client=gateway_client)
 
     # docs-1: end
-    address = deploy_proxy_to_contract_oz_argent.deployed_contract.address
+    address = proxy_oz_argent_eth.deployed_contract.address
     # docs-1: start
     # To use contract behind a proxy as a regular contract, set proxy_config to True
-    # It will check if your proxy is OpenZeppelin or ArgentX proxy
+    # It will check if your proxy is OpenZeppelin proxy / ArgentX proxy / proxy of Starknet Eth contract
     contract = await Contract.from_address(
         address=address, client=gateway_client, proxy_config=True
     )
 
     # After that contract can be used as usual
     # docs-1: end
     # docs-2: start
-    # To resolve proxy contract other than OpenZeppelin or ArgentX, a custom ProxyCheck is needed
+    # To resolve proxy contract other than OpenZeppelin / ArgentX / Starknet Eth proxy, a custom ProxyCheck is needed
```

Comment:
Maybe it should be ERC20 proxy or sth, but I wanted to keep the names consistent with the ProxyCheck name. Would any of StarknetEthProxyCheck / StarknetERC20ProxyCheck / ERC20ProxyCheck be better?